eva-server/src/main/java/uk/ac/ebi/eva/server/ws/GeneWSServerV2.java

```diff
@@ -59,13 +60,23 @@ public GeneWSServerV2() {
     }
 
     @GetMapping(value = "/{geneIds}/variants")
-    public ResponseEntity getVariantsByGene(@PathVariable("geneIds") List<String> geneIds,
-                                            @RequestParam(name = "species") String species,
-                                            @RequestParam(name = "assembly") String assembly,
-                                            @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                            @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                            HttpServletResponse response,
-                                            @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByGene(
+            @ApiParam(value = "Comma separated values of gene names or gene ids. e.g. BRCA2,FOXP2,ENSG00000223972")
+            @PathVariable("geneIds") List<String> geneIds,
+            @ApiParam(value = "First letter of the genus, followed by the full species name, e.g. hsapiens. Allowed" +
+                    " values can be looked up in /v1/meta/species/list/ in the field named 'taxonomyCode'.",
+                    required = true)
+            @RequestParam(name = "species") String species,
+            @ApiParam(value = "Encoded assembly name, e.g. grch37. Allowed values can be looked up in " +
+                    "/v1/meta/species/list/ in the field named 'assemblyCode'.", required = true)
+            @RequestParam(name = "assembly") String assembly,
+            @ApiParam(value = "The number of the page that shoulde be displayed. Starts from 0 and is an integer." +
+                    " e.g. 0")
+            @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
+            @ApiParam(value = "The number of elements that should be displayed in a single page. e.g. 5")
```

Comment:
No need to add the last part because a default value will be displayed. The fact that they don't match could make it confusing.
```suggestion
            @ApiParam(value = "The number of elements that should be retrieved per page.")
```
Using `retrieved` instead of `displayed` because we are talking about an API, not necessarily about visualization.